inkscape/svg2shenzhen/prepare.py

```diff
@@ -62,19 +62,19 @@ def setDocumentSquare(self, width, height):
 
     def createLayer(self, layer_name):
         svg = self.document.xpath('//svg:svg',namespaces=inkex.NSS)[0]
-        layer = inkex.etree.SubElement(svg, 'g')
+        layer = etree.SubElement(svg, 'g')
```

Comment:
These three lines can be replaced with `layer = svg.add(Layer(layer_name))` the API does a lot of this work for you.

Reply:
I'm not familiar with the Inkscape APIs. I'm not comfortable making changes that may have side effects I'm not aware of.